.drone.yml

```diff
@@ -15,32 +15,36 @@ pipeline:
       - GOOS=windows GOARCH=amd64 CGO_ENABLED=0 go build -o bin/kube-cfn-signal_windows_amd64.exe
 
   build_docker_image:
-    image: docker:1.12
+    image: docker:17.12.0
     environment:
-      - DOCKER_HOST=tcp://127.0.0.1:2375
+      - DOCKER_HOST=tcp://172.17.0.1:2375
     commands:
       - docker build -t image .
```

Comment:
you probably want to build that images with $${DRONE_COMMIT_SHA}

Reply:
will do